src/CleanUpModelsCommand.php

```diff
@@ -112,7 +112,13 @@ protected function getAllModelsFromEachDirectory(array $directories) : Collectio
 
     protected function getClassNamesInDirectory(string $directory) : Collection
     {
-        return collect($this->filesystem->allFiles($directory))->map(function ($path) {
+        if (config('model-cleanup.recursive', true)) {
+            $files = $this->filesystem->allFiles($directory);
+        } else {
```

Comment:
We try to avoid `else` statements as much as possible. Could you use a ternary here?